dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -471,7 +485,7 @@ public void updateVersionMetadata(String filePath, Version<?> version, Descripto
                 LOG.info(message);
             }
             if (version.getReference() != null) {
-                String readmeContent = getREADMEContent(repositoryId, version.getReference());
+                String readmeContent = getREADMEContent(repositoryId, version.getReference(), version.getReadMePath());
```

Comment:
Similar to Flynn's comment above, this might be a good opportunity to pick one capitalization of "README" and make everything as consistent as possible (within coding guidelines).

Reply:
I'm ok with whatever, got any preferences?
Note that whatever we pick will show up in the .dockstore.yml

Reply:
Also tagging @coverbeck @fhembroff  @aofarrel for opinions